app/models/application_form.rb

```diff
@@ -21,6 +21,7 @@ class ApplicationForm < ApplicationRecord
   MINIMUM_COMPLETE_REFERENCES = 2
   MAXIMUM_REFERENCES = 10
   EQUALITY_AND_DIVERSITY_MINIMAL_ATTR = %w[sex disabilities ethnic_group].freeze
+  ENGLISH_SPEAKING_NATIONALITIES = %w[British Irish].freeze
```

Comment:
I think you will also need to include English, Welsh, Scottish and Northern Irish.

Reply:
@davidgisbey yep, handled this in a [later commit](https://github.com/DFE-Digital/apply-for-teacher-training/pull/2474/commits/5075136f2548b3c08874530da99cd9f65fa3aaa0), by switching to country codes instead of the full name.